StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -51,19 +51,38 @@ extension PaymentSheet {
             /// A user facing string representing the payment method; e.g. "Apple Pay" or "····4242" for a card
             public let label: String
 
+            /// The billing details associated with the customer's desired payment option
```

Comment:
To be consistent with the other docstrings, which say "payment method" and not "payment option"
```suggestion
            /// The billing details associated with the customer's desired payment method
```